app/channel_test.go

```diff
@@ -25,21 +25,17 @@ func TestPermanentDeleteChannel(t *testing.T) {
 	})
 
 	channel, err := th.App.CreateChannel(&model.Channel{DisplayName: "deletion-test", Name: "deletion-test", Type: model.CHANNEL_OPEN, TeamId: th.BasicTeam.Id}, false)
-	if err != nil {
-		t.Fatal(err.Error())
-	}
+	require.Nil(t, err)
 	defer func() {
 		th.App.PermanentDeleteChannel(channel)
 	}()
 
 	incoming, err := th.App.CreateIncomingWebhookForChannel(th.BasicUser.Id, channel, &model.IncomingWebhook{ChannelId: channel.Id})
-	if err != nil {
-		t.Fatal(err.Error())
-	}
+	require.Nil(t, err)
 	defer th.App.DeleteIncomingWebhook(incoming.Id)
 
 	if incoming, err = th.App.GetIncomingWebhook(incoming.Id); incoming == nil || err != nil {
```

Comment:
why not doing this?
```
incoming, err = th.App.GetIncomingWebhook(incoming.Id)
require.NotNil(t, incoming, "Incoming should not be nil")
require.Nil(t, err, "Unable to get new incoming webhook")
```
same for the others.

Reply:
@cpanato 
My apologies on the above issue, I thought I had fixed them, looks like I did but never committed it. 

Should be ready now.